loopd/view.go

```diff
@@ -64,13 +64,18 @@ func viewOut(swapClient *loop.Client, chainParams *chaincfg.Params) error {
 		fmt.Printf("   Preimage: %v\n", s.Contract.Preimage)
 		fmt.Printf("   Htlc address: %v\n", htlc.Address)
 
-		unchargeChannel := "any"
-		if s.Contract.UnchargeChannel != nil {
-			unchargeChannel = strconv.FormatUint(
-				*s.Contract.UnchargeChannel, 10,
-			)
+		var unchargeChannels string
+		if len(s.Contract.OutgoingChanSet) > 0 {
+			for i, chanID := range s.Contract.OutgoingChanSet {
+				if i > 0 {
+					unchargeChannels += ", "
+				}
+				unchargeChannels += strconv.FormatUint(chanID, 10)
```

Comment:
Same here, maybe make a short helper func which we can just call inline in the log since we do this more than once? Th

Reply:
helper added, see above